vueapp/src/views/TilePad.vue

```diff
@@ -1,66 +1,72 @@
 <template>
 	<div>
-		<Sentence
-			v-if="sentenceMode && !editMode"
-			:tile-pad-to-display="sentenceTiles"
-			@clearSentence="sentenceTiles = []"
-			@speakSentence="speakSentence"
-			@removeFromSentence="sentenceTiles.splice($event, 1)"
-		/>
-		<edit-mode-header
-			v-if="editMode"
-		/>
-		<v-container
-			fluid
-			class="grey lighten-5 pb-10"
-			style="{text-align: center}"
-		>
-			<v-row dense>
-				<Tile
-					v-if="editMode"
-					id="newTile"
-					:new-tile="true"
-					:tile-page="currentTilePadPage"
-				/>
+		<vue-custom-scrollbar class="scroll-area" :settings="settings">
```

Comment:
Hmm out of curiosity, why did you wrap the TilePad into a `vue-custom-scrollbar`?

Reply:
I wished to use a sleeker scrollbar, but it seems that it didn't work. I guess we can remove it.